inc/Admin/Fields/Checkbox.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace AntispamBee\Admin\Fields;
+
+use AntispamBee\Admin\RenderElement;
+
+/**
+ * Checkbox field.
+ */
+class Checkbox extends Field implements RenderElement {
+    /**
+     * Get HTML.
+     */
+    public function render() {
+        echo '<input type="checkbox" name="' . $this->get_name() . '" value="1" ' . checked( 1, $this->get_value(), true ) . ' />';
```

Comment:
I would like us to use `printf` instead of string concats to make the code more reable:

```php
printf(
	'<input type="checkbox" name="%s" value="1" %s />',
	$this->get_name(),
	checked( 1, $this->get_value(), false )
);
```